apps/api/v2/src/modules/organizations/services/organizations-users-service.ts

```diff
@@ -0,0 +1,116 @@
+import { EmailService } from "@/modules/email/email.service";
+import { CreateOrganizationUserInput } from "@/modules/organizations/inputs/create-organization-user.input";
+import { UpdateOrganizationUserInput } from "@/modules/organizations/inputs/update-organization-user.input";
+import { OrganizationsUsersRepository } from "@/modules/organizations/repositories/organizations-users.repository";
+import { CreateUserInput } from "@/modules/users/inputs/create-user.input";
+import { Injectable, ConflictException } from "@nestjs/common";
+import { plainToInstance } from "class-transformer";
+
+import {
+  createNewUsersConnectToOrgIfExists,
+  sendSignupToOrganizationEmail,
+  getTranslation,
+} from "@calcom/platform-libraries";
+import { Team } from "@calcom/prisma/client";
+
+@Injectable()
+export class OrganizationsUsersService {
+  constructor(
+    private readonly organizationsUsersRepository: OrganizationsUsersRepository,
+    private readonly emailService: EmailService
+  ) {}
+
+  async getUsers(orgId: number, emailInput?: string | string[]) {
+    const emailArray = !emailInput ? [] : Array.isArray(emailInput) ? emailInput : [emailInput];
+
+    const users = await this.organizationsUsersRepository.getOrganizationUsers(orgId, emailArray);
+
+    return users;
+  }
```

Comment:
I'd prefer if we didn't have to deal with inputs format outside of class validator, this is where we should already transform a string email to an array of one email if necessary